db/schema.rb

```diff
@@ -9,1324 +8,1301 @@
 # from scratch. The latter is a flawed and unsustainable approach (the more migrations
 # you'll amass, the slower it'll run and the greater likelihood for issues).
 #
-# It's strongly recommended to check this file into your version control system.
+# It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(:version => 20160706031054) do
+ActiveRecord::Schema.define(version: 20190611212339) do
```

Comment:
If we bump the version here, we need to check the existing PRs for migrations and bump their timestamps to later.

Reply:
I don't think that's a problem -- we originally created the Has Migrations label to remind us to check the migration date and potentially update it following a schema change.

Reply:
Left a comment on the two PRs that will need updating.